recipes/abayesqr/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: abayesqr
+  version: "1.0"
+
+source:
+  fn: master.zip
+  url: https://github.com/SoYeonA/aBayesQR/archive/master.zip
```

Comment:
Can you please track a release here or a specific commit.

Reply:
What I meant is more like this: https://github.com/SoYeonA/aBayesQR/archive/37d2e6c83e571671349a2c65a302d0228dcb8921.tar.gz